packages/mds-policy/package.json

```diff
@@ -29,8 +29,8 @@
     "test": "yarn test:prettier && yarn test:eslint && yarn test:unit",
     "test:eslint": "eslint --ignore-path ../../.gitignore '**/*.ts'",
     "test:prettier": "prettier --write --check --ignore-path ../../.gitignore '**/*.ts'",
-    "test:unit": "nyc --check-coverage --exclude tests --extension .ts --lines 80 --reporter=text --reporter=html ts-mocha --require source-map-support/register --recursive tests/**/*.ts",
+    "test:unit": "nyc --check-coverage --exclude tests --extension .ts --lines 79 --reporter=text --reporter=html ts-mocha --require source-map-support/register --recursive tests/**/*.ts",
```

Comment:
y

Reply:
Added some moar tests to improve coverage back to its previous glory!